### DISCUSSION TITLE
Storing durations: Interval Data Type?

### DISCUSSION THREAD

**Discussion by jtkeyva** [supabase/supabase#7529] [Questions] [answered]
Hello, I need to store the duration as well as start and stop time for videos and audio. How do I set up a data type for Interval?

Thanks
https://www.geeksforgeeks.org/postgresql-interval-data-type/

**Comment by steve-chavez** [comment, marked as answer]
It should work with plain SQL:
```sql
create table mytbl( id int, duration interval );
```

**Reply by jtkeyva** [reply]
thanks, so just not part of the native types built in to Supabase Studio. makes sense

**Comment by proton1k** [comment]
I'd vote up for including Postgres INTERVAL into the available data types in the ui 👍 (if there's a voting...)

**Comment by rookledookle** [comment]
Upvote!

**Comment by alex-shortt** [comment]
bump
